src/test/java/redis/clients/jedis/tests/ShardedJedisTest.java

```diff
@@ -270,9 +270,9 @@ public void testMasterSlaveShardingConsistency() {
         Hashing.MURMUR_HASH);
 
     List<JedisShardInfo> otherShards = new ArrayList<JedisShardInfo>(3);
-    otherShards.add(new JedisShardInfo("otherhost", Protocol.DEFAULT_PORT));
-    otherShards.add(new JedisShardInfo("otherhost", Protocol.DEFAULT_PORT + 1));
-    otherShards.add(new JedisShardInfo("otherhost", Protocol.DEFAULT_PORT + 2));
+    otherShards.add(new JedisShardInfo("127.0.0.1", Protocol.DEFAULT_PORT));
```

Comment:
why did you change all the host names to IPs?

Reply:
@gkorland https://github.com/redis/jedis/pull/2368#issuecomment-790262400